src/components/Place/PlaceDescription/PlaceDescription.tsx

```diff
@@ -28,8 +27,8 @@ export type PlaceDescriptionProps = {
   loadingFavorite?: boolean
 }
 
-export default React.memo(function PlaceDescription(
-  props: PlaceDescriptionProps
+export default React.memo(function LocationDescription(
```

Comment:
This show be renamed to `PlaceDescription` as the File name is `PlaceDescription`